lua/neogit/lib/finder.lua

```diff
@@ -1,9 +1,10 @@
 local config = require("neogit.config")
-local async = require("plenary.async")
```

Comment:
I changed these back to `a` to be consistent with the rest of the project.

Reply:
The issue with `a` is that is can be confused with `local a = vim.api` so the names clash when moving code around.

I think being slighly longer with `async` and `api` to not confuse them in shortening is a good idea.

Reply:
Totally fair - in that case, why don't we do a project-wide update after all this is merged?